microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -249,7 +265,9 @@ If you have an offline deployment scenario, where app deployment has to work ent
    ```
 
 
+<!-- ------------------------------ -->
 #### Test your app for forward-compatibility
+<!-- todo: check this section for any needed changes -->
```

Comment:
check this section for any needed changes - "Test your app for forward-compatibility", in distribution.md